packages/runner-ct/src/SpecList/components/SearchSpec.scss

```diff
@@ -7,7 +7,9 @@
   width: 100%;
   padding: 8px;
   box-sizing: border-box;
-
+  background-color: white;
+  z-index: 10;
```

Comment:
Unless you have a `z-index` of 9, this is an incorrect assignment. `z-index` is entirely based on [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context), so if you're an old-school JS dev, setting `z-index: 1000` or whatever didn't actually do anything special.

Most likely (without pulling down the branch and trying it myself) you want `z-index: 1`

Reply:
No. You can pull the branch and debug it yourself. I prefer tailwind style of indexing with a 10x. Previous is 1 new is 10 next is 100